code/modules/reagents/recipes.dm

```diff
@@ -1827,47 +1826,47 @@
 
 /datum/chemical_reaction/noexcutite
 	result = "noexcutite"
-	required_reagents = list("oxycodone" = 1, "anti_toxin" = 1)
+	required_reagents = list("tramadol" = 1, "anti_toxin" = 1)
```

Comment:
Not crazy about this. Interferes with medical treatment. Recommend requiring a catalyst.

Reply:
oh true thing thanks. forgot it could mix inside bodies

Reply:
changes made

Reply:
Good stuff